pandas/tests/frame/test_analytics.py

```diff
@@ -241,24 +241,16 @@ def test_bool_describe_in_mixed_frame(self):
             'int_data': [10, 20, 30, 40, 50],
         })
 
-        # Boolean data and integer data is included in .describe() output,
-        # string data isn't
-        self.assert_numpy_array_equal(df.describe().columns, [
-                                      'bool_data', 'int_data'])
+        # Integer data are included in .describe() output,
```

Comment:
ideally construct a comparison frame here and use ``assert_frame_equal``.

add the issue number as a comment